File: java/src/main/java/com/languageweaver/sdk/samples/dictionaries/CloudDictionaryService.java
package com.languageweaver.sdk.samples.dictionaries;

import com.languageweaver.sdk.common.cloud.CloudLanguageWeaverClient;
import com.languageweaver.sdk.translate.cloud.result.CloudDictionariesResult;

public class CloudDictionaryService {
    public static final int PAGE_NUMBER = 1;
    public static final int PAGE_SIZE = 100;

    public static final int PAGE_NUMBER = 1;
    public static final int PAGE_SIZE = 100;

    public static void main(String[] args) throws Exception {
        try (CloudLanguageWeaverClient lwClient = new CloudLanguageWeaverClient().build()) {
            final CloudDictionariesResult dictionariesResult = lwClient.getCloudDictionaries(PAGE_NUMBER, PAGE_SIZE);

            if (dictionariesResult.getDictionaries() != null) {
                dictionariesResult.getDictionaries()
                        .forEach(dictionary -> System.out.println(dictionary.getDictionaryId()));
            }
        }
    }
}

File: dotnet/Samples/Dictionaries/CloudDictionaryService.cs
using LanguageWeaver.Sdk.Common.Cloud;
using LanguageWeaver.Sdk.Translate.Cloud.Result;

namespace LanguageWeaver.Sdk.Samples.Dictionaries;

public static class CloudDictionaryService
{
    public static readonly int PAGE_NUMBER = 1;
    public static readonly int PAGE_SIZE = 100;

    public static void Main()
    {
        using var lwClient = new CloudLanguageWeaverClient().Build();
        CloudDictionariesResult dictionariesResult = lwClient.GetCloudDictionaries(PAGE_NUMBER, PAGE_SIZE);

        dictionariesResult.Dictionaries
            .ForEach(dictionary => Console.WriteLine(dictionary.DictionaryId));
    }
}